sdcm/gemini_thread.py

```diff
@@ -57,25 +58,20 @@ def run(self):
                     break
 
 
-class GeminiStressThread:  # pylint: disable=too-many-instance-attributes
+class GeminiStressThread(DockerBasedStressThread):  # pylint: disable=too-many-instance-attributes
 
-    def __init__(self, test_cluster, oracle_cluster, loaders, gemini_cmd, timeout=None, outputdir=None, params=None):  # pylint: disable=too-many-arguments
-        self.loaders = loaders
+    def __init__(self, test_cluster, oracle_cluster, loaders, gemini_cmd, timeout=None, params=None):  # pylint: disable=too-many-arguments
+        super().__init__(loader_set=loaders, stress_cmd=gemini_cmd, timeout=timeout, params=params)
         self.gemini_cmd = gemini_cmd
```

Comment:
May we don't need  gemini_cmd? because we already inherint stress_cmd property and could operate with it

Reply:
yeah we could, I was trying to minimize the change across to few as possible.
but yes, that can be aligned

also the configuration value for it is called `gemini_cmd`, which kind of add confusion. but yes it can be aligned

Reply:
it's basically what the code does today for longevity and such
https://github.com/scylladb/scylla-cluster-tests/blob/e0d94702f8706951532b2010735c76ef3e2d3bfe/sdcm/tester.py#L1613

gemini happens to have it's own test class and reporter so it's not using this code. and calls `run_gemini_thread` directly.